From 9b59765e3ac41e62a3d98e9ae6030d7fa7726e27 Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Wed, 23 Oct 2024 17:09:07 +0000
Subject: [PATCH 101/128] UBUNTU: [Config] Optimize defconfig

- Remove unwanted Display panel configs
- Remove unnecessary Frame buffer configs
- Enabled DEBUG_INFO configs
- Added changes for USB Boot. Configure USB_MASS_STORAGE as inbuilt

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
---
 debian/config/annotations | 805 +++++++++++++++-----------------------
 1 file changed, 325 insertions(+), 480 deletions(-)

diff --git a/debian/config/annotations b/debian/config/annotations
index db25e43dfa27..c8507f626d04 100644
--- a/debian/config/annotations
+++ b/debian/config/annotations
@@ -514,7 +514,6 @@ CONFIG_9P_FSCACHE                               policy<{'riscv64': 'y'}>
 CONFIG_9P_FS_POSIX_ACL                          policy<{'riscv64': 'y'}>
 CONFIG_9P_FS_SECURITY                           policy<{'riscv64': 'y'}>
 CONFIG_ABP060MG                                 policy<{'riscv64': 'n'}>
-CONFIG_AC97_BUS                                 policy<{'riscv64': 'm'}>
 CONFIG_ACENIC                                   policy<{'riscv64': 'm'}>
 CONFIG_ACENIC_OMIT_TIGON_I                      policy<{'riscv64': 'n'}>
 CONFIG_ACORN_PARTITION                          policy<{'riscv64': 'n'}>
@@ -759,7 +758,6 @@ CONFIG_ARCNET_COM90xx                           policy<{'riscv64': 'm'}>
 CONFIG_ARCNET_COM90xxIO                         policy<{'riscv64': 'm'}>
 CONFIG_ARCNET_RAW                               policy<{'riscv64': 'm'}>
 CONFIG_ARCNET_RIM_I                             policy<{'riscv64': 'm'}>
-CONFIG_ARCX_ANYBUS_CONTROLLER                   policy<{'riscv64': 'm'}>
 CONFIG_ARM_SMMU_V3                              policy<{'riscv64': 'y'}>
 CONFIG_ARM_SMMU_V3_SVA                          policy<{'riscv64': 'n'}>
 CONFIG_AS3935                                   policy<{'riscv64': 'n'}>
@@ -1651,10 +1649,17 @@ CONFIG_DEBUG_FS_ALLOW_ALL                       policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_FS_ALLOW_NONE                      policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_FS_DISALLOW_MOUNT                  policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_GPIO                               policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO                               policy<{'riscv64': 'y'}>
+CONFIG_DEBUG_INFO_BTF                           policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO_COMPRESSED_NONE               policy<{'riscv64': 'y'}>
+CONFIG_DEBUG_INFO_COMPRESSED_ZLIB               policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO_COMPRESSED_ZSTD               policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_INFO_DWARF4                        policy<{'riscv64': 'n'}>
-CONFIG_DEBUG_INFO_DWARF5                        policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO_DWARF5                        policy<{'riscv64': 'y'}>
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT       policy<{'riscv64': 'n'}>
-CONFIG_DEBUG_INFO_NONE                          policy<{'riscv64': 'y'}>
+CONFIG_DEBUG_INFO_NONE                          policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO_REDUCED                       policy<{'riscv64': 'n'}>
+CONFIG_DEBUG_INFO_SPLIT                         policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_IRQFLAGS                           policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_KERNEL_DC                          policy<{'riscv64': 'n'}>
 CONFIG_DEBUG_KMEMLEAK                           policy<{'riscv64': 'n'}>
@@ -1867,15 +1872,15 @@ CONFIG_DRM_I2C_SIL164                           policy<{'riscv64': 'n'}>
 CONFIG_DRM_IMG                                  policy<{'riscv64': 'y'}>
 CONFIG_DRM_IMG_VOLCANIC                         policy<{'riscv64': 'm'}>
 CONFIG_DRM_ITE_IT6505                           policy<{'riscv64': 'n'}>
-CONFIG_DRM_ITE_IT66121                          policy<{'riscv64': 'm'}>
+CONFIG_DRM_ITE_IT66121                          policy<{'riscv64': 'n'}>
 CONFIG_DRM_KMS_HELPER                           policy<{'riscv64': 'y'}>
 CONFIG_DRM_KOMEDA                               policy<{'riscv64': 'n'}>
 CONFIG_DRM_LOAD_EDID_FIRMWARE                   policy<{'riscv64': 'y'}>
 CONFIG_DRM_LOGICVC                              policy<{'riscv64': 'm'}>
-CONFIG_DRM_LONTIUM_LT8912B                      policy<{'riscv64': 'm'}>
-CONFIG_DRM_LONTIUM_LT9211                       policy<{'riscv64': 'm'}>
-CONFIG_DRM_LONTIUM_LT9611                       policy<{'riscv64': 'm'}>
-CONFIG_DRM_LONTIUM_LT9611UXC                    policy<{'riscv64': 'm'}>
+CONFIG_DRM_LONTIUM_LT8912B                      policy<{'riscv64': 'n'}>
+CONFIG_DRM_LONTIUM_LT9211                       policy<{'riscv64': 'n'}>
+CONFIG_DRM_LONTIUM_LT9611                       policy<{'riscv64': 'n'}>
+CONFIG_DRM_LONTIUM_LT9611UXC                    policy<{'riscv64': 'n'}>
 CONFIG_DRM_LOONGSON                             policy<{'riscv64': 'm'}>
 CONFIG_DRM_LVDS_CODEC                           policy<{'riscv64': 'm'}>
 CONFIG_DRM_MEGACHIPS_STDPXXXX_GE_B850V3_FW      policy<{'riscv64': 'n'}>
@@ -1941,17 +1946,15 @@ CONFIG_DRM_PANEL_RONBO_RB070D30                 policy<{'riscv64': 'n'}>
 CONFIG_DRM_PANEL_SAMSUNG_ATNA33XC20             policy<{'riscv64': 'n'}>
 CONFIG_DRM_PANEL_SAMSUNG_DB7430                 policy<{'riscv64': 'n'}>
 CONFIG_DRM_PANEL_SAMSUNG_LD9040                 policy<{'riscv64': 'n'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6D16D0                policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6D27A1                policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6D7AA0                policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E3HA2                policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E63J0X03             policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E63M0                policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E63M0_DSI            policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E63M0_SPI            policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E88A0_AMS452EF01     policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_S6E8AA0                policy<{'riscv64': 'm'}>
-CONFIG_DRM_PANEL_SAMSUNG_SOFEF00                policy<{'riscv64': 'm'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6D16D0                policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6D27A1                policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6D7AA0                policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6E3HA2                policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6E63J0X03             policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6E63M0                policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6E88A0_AMS452EF01     policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_S6E8AA0                policy<{'riscv64': 'n'}>
+CONFIG_DRM_PANEL_SAMSUNG_SOFEF00                policy<{'riscv64': 'n'}>
 CONFIG_DRM_PANEL_SEIKO_43WVF1G                  policy<{'riscv64': 'n'}>
 CONFIG_DRM_PANEL_SHARP_LQ101R1SX01              policy<{'riscv64': 'n'}>
 CONFIG_DRM_PANEL_SHARP_LS037V7DW01              policy<{'riscv64': 'n'}>
@@ -2179,7 +2182,7 @@ CONFIG_FB_3DFX                                  policy<{'riscv64': 'm'}>
 CONFIG_FB_3DFX_ACCEL                            policy<{'riscv64': 'n'}>
 CONFIG_FB_3DFX_I2C                              policy<{'riscv64': 'n'}>
 CONFIG_FB_ARK                                   policy<{'riscv64': 'm'}>
-CONFIG_FB_ASILIANT                              policy<{'riscv64': 'y'}>
+CONFIG_FB_ASILIANT                              policy<{'riscv64': 'n'}>
 CONFIG_FB_ATY                                   policy<{'riscv64': 'm'}>
 CONFIG_FB_ATY128                                policy<{'riscv64': 'm'}>
 CONFIG_FB_ATY128_BACKLIGHT                      policy<{'riscv64': 'y'}>
@@ -2193,10 +2196,9 @@ CONFIG_FB_CARMINE_DRAM_EVAL                     policy<{'riscv64': 'y'}>
 CONFIG_FB_CFB_COPYAREA                          policy<{'riscv64': 'y'}>
 CONFIG_FB_CFB_FILLRECT                          policy<{'riscv64': 'y'}>
 CONFIG_FB_CFB_IMAGEBLIT                         policy<{'riscv64': 'y'}>
-CONFIG_FB_CIRRUS                                policy<{'riscv64': 'm'}>
+CONFIG_FB_CIRRUS                                policy<{'riscv64': 'n'}>
 CONFIG_FB_CORE                                  policy<{'riscv64': 'y'}>
-CONFIG_FB_CYBER2000                             policy<{'riscv64': 'm'}>
-CONFIG_FB_CYBER2000_DDC                         policy<{'riscv64': 'y'}>
+CONFIG_FB_CYBER2000                             policy<{'riscv64': 'n'}>
 CONFIG_FB_DDC                                   policy<{'riscv64': 'm'}>
 CONFIG_FB_DEFERRED_IO                           policy<{'riscv64': 'y'}>
 CONFIG_FB_DEVICE                                policy<{'riscv64': 'y'}>
@@ -2206,7 +2208,7 @@ CONFIG_FB_FOREIGN_ENDIAN                        policy<{'riscv64': 'n'}>
 CONFIG_FB_GOLDFISH                              policy<{'riscv64': 'm'}>
 CONFIG_FB_I740                                  policy<{'riscv64': 'm'}>
 CONFIG_FB_IBM_GXT4500                           policy<{'riscv64': 'n'}>
-CONFIG_FB_IMSTT                                 policy<{'riscv64': 'y'}>
+CONFIG_FB_IMSTT                                 policy<{'riscv64': 'n'}>
 CONFIG_FB_IOMEM_HELPERS                         policy<{'riscv64': 'y'}>
 CONFIG_FB_KYRO                                  policy<{'riscv64': 'm'}>
 CONFIG_FB_MATROX                                policy<{'riscv64': 'm'}>
@@ -2227,8 +2229,7 @@ CONFIG_FB_NVIDIA_BACKLIGHT                      policy<{'riscv64': 'y'}>
 CONFIG_FB_NVIDIA_DEBUG                          policy<{'riscv64': 'n'}>
 CONFIG_FB_NVIDIA_I2C                            policy<{'riscv64': 'y'}>
 CONFIG_FB_OPENCORES                             policy<{'riscv64': 'm'}>
-CONFIG_FB_PM2                                   policy<{'riscv64': 'm'}>
-CONFIG_FB_PM2_FIFO_DISCONNECT                   policy<{'riscv64': 'y'}>
+CONFIG_FB_PM2                                   policy<{'riscv64': 'n'}>
 CONFIG_FB_PM3                                   policy<{'riscv64': 'm'}>
 CONFIG_FB_RADEON                                policy<{'riscv64': 'm'}>
 CONFIG_FB_RADEON_BACKLIGHT                      policy<{'riscv64': 'y'}>
@@ -2238,7 +2239,7 @@ CONFIG_FB_RIVA                                  policy<{'riscv64': 'm'}>
 CONFIG_FB_RIVA_BACKLIGHT                        policy<{'riscv64': 'y'}>
 CONFIG_FB_RIVA_DEBUG                            policy<{'riscv64': 'n'}>
 CONFIG_FB_RIVA_I2C                              policy<{'riscv64': 'y'}>
-CONFIG_FB_S1D13XXX                              policy<{'riscv64': 'm'}>
+CONFIG_FB_S1D13XXX                              policy<{'riscv64': 'n'}>
 CONFIG_FB_S3                                    policy<{'riscv64': 'm'}>
 CONFIG_FB_S3_DDC                                policy<{'riscv64': 'y'}>
 CONFIG_FB_SAVAGE                                policy<{'riscv64': 'm'}>
@@ -2270,7 +2271,7 @@ CONFIG_FB_VT8623                                policy<{'riscv64': 'm'}>
 CONFIG_FDDI                                     policy<{'riscv64': 'y'}>
 CONFIG_FEALNX                                   policy<{'riscv64': 'm'}>
 CONFIG_FIB_RULES                                policy<{'riscv64': 'y'}>
-CONFIG_FIELDBUS_DEV                             policy<{'riscv64': 'm'}>
+CONFIG_FIELDBUS_DEV                             policy<{'riscv64': 'n'}>
 CONFIG_FILE_LOCKING                             policy<{'riscv64': 'y'}>
 CONFIG_FIND_BIT_BENCHMARK                       policy<{'riscv64': 'n'}>
 CONFIG_FIPS_SIGNATURE_SELFTEST                  policy<{'riscv64': 'n'}>
@@ -2343,7 +2344,6 @@ CONFIG_FUTEX                                    policy<{'riscv64': 'y'}>
 CONFIG_FUTEX_PI                                 policy<{'riscv64': 'y'}>
 CONFIG_FWNODE_MDIO                              policy<{'riscv64': 'y'}>
 CONFIG_FW_CACHE                                 policy<{'riscv64': 'y'}>
-CONFIG_FW_CS_DSP                                policy<{'riscv64': 'm'}>
 CONFIG_FW_DEVLINK_SYNC_STATE_TIMEOUT            policy<{'riscv64': 'n'}>
 CONFIG_FW_LOADER                                policy<{'riscv64': 'y'}>
 CONFIG_FW_LOADER_COMPRESS                       policy<{'riscv64': 'y'}>
@@ -2369,6 +2369,7 @@ CONFIG_GCC_ASM_GOTO_OUTPUT_WORKAROUND           policy<{'riscv64': 'y'}>
 CONFIG_GCC_SUPPORTS_DYNAMIC_FTRACE              policy<{'riscv64': 'y'}>
 CONFIG_GCC_VERSION                              policy<{'riscv64': '130200'}>
 CONFIG_GCOV_KERNEL                              policy<{'riscv64': 'n'}>
+CONFIG_GDB_SCRIPTS                              policy<{'riscv64': 'y'}>
 CONFIG_GEMINI_ETHERNET                          policy<{'riscv64': 'm'}>
 CONFIG_GENERIC_ADC_BATTERY                      policy<{'riscv64': 'n'}>
 CONFIG_GENERIC_ADC_THERMAL                      policy<{'riscv64': 'n'}>
@@ -2482,27 +2483,7 @@ CONFIG_GPIO_XILINX                              policy<{'riscv64': 'n'}>
 CONFIG_GPIO_XRA1403                             policy<{'riscv64': 'n'}>
 CONFIG_GP_PCI1XXXX                              policy<{'riscv64': 'm'}>
 CONFIG_GRACE_PERIOD                             policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS                                  policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_AUDIO                            policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_AUDIO_APB_CODEC                  policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_BOOTROM                          policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_BRIDGED_PHY                      policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_ES2                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_FIRMWARE                         policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_GPIO                             policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_HID                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_I2C                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_LIGHT                            policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_LOG                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_LOOPBACK                         policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_POWER                            policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_PWM                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_RAW                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_SDIO                             policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_SPI                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_UART                             policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_USB                              policy<{'riscv64': 'm'}>
-CONFIG_GREYBUS_VIBRATOR                         policy<{'riscv64': 'm'}>
+CONFIG_GREYBUS                                  policy<{'riscv64': 'n'}>
 CONFIG_GRO_CELLS                                policy<{'riscv64': 'y'}>
 CONFIG_GTP                                      policy<{'riscv64': 'm'}>
 CONFIG_GUP_TEST                                 policy<{'riscv64': 'n'}>
@@ -2725,8 +2706,6 @@ CONFIG_HISI_HIKEY_USB                           policy<{'riscv64': 'n'}>
 CONFIG_HMC425                                   policy<{'riscv64': 'n'}>
 CONFIG_HMC6352                                  policy<{'riscv64': 'm'}>
 CONFIG_HMM_MIRROR                               policy<{'riscv64': 'y'}>
-CONFIG_HMS_ANYBUSS_BUS                          policy<{'riscv64': 'm'}>
-CONFIG_HMS_PROFINET                             policy<{'riscv64': 'm'}>
 CONFIG_HOSTAP                                   policy<{'riscv64': 'm'}>
 CONFIG_HOSTAP_FIRMWARE                          policy<{'riscv64': 'y'}>
 CONFIG_HOSTAP_FIRMWARE_NVRAM                    policy<{'riscv64': 'y'}>
@@ -2951,11 +2930,11 @@ CONFIG_INPUT_EVDEV                              policy<{'riscv64': 'y'}>
 CONFIG_INPUT_FF_MEMLESS                         policy<{'riscv64': 'm'}>
 CONFIG_INPUT_JOYDEV                             policy<{'riscv64': 'm'}>
 CONFIG_INPUT_JOYSTICK                           policy<{'riscv64': 'n'}>
-CONFIG_INPUT_KEYBOARD                           policy<{'riscv64': 'y'}>
+CONFIG_INPUT_KEYBOARD                           policy<{'riscv64': 'n'}>
 CONFIG_INPUT_LEDS                               policy<{'riscv64': 'm'}>
 CONFIG_INPUT_MATRIXKMAP                         policy<{'riscv64': 'm'}>
 CONFIG_INPUT_MISC                               policy<{'riscv64': 'n'}>
-CONFIG_INPUT_MOUSE                              policy<{'riscv64': 'y'}>
+CONFIG_INPUT_MOUSE                              policy<{'riscv64': 'n'}>
 CONFIG_INPUT_MOUSEDEV                           policy<{'riscv64': 'y'}>
 CONFIG_INPUT_MOUSEDEV_PSAUX                     policy<{'riscv64': 'y'}>
 CONFIG_INPUT_MOUSEDEV_SCREEN_X                  policy<{'riscv64': '1024'}>
@@ -2963,7 +2942,6 @@ CONFIG_INPUT_MOUSEDEV_SCREEN_Y                  policy<{'riscv64': '768'}>
 CONFIG_INPUT_SPARSEKMAP                         policy<{'riscv64': 'm'}>
 CONFIG_INPUT_TABLET                             policy<{'riscv64': 'n'}>
 CONFIG_INPUT_TOUCHSCREEN                        policy<{'riscv64': 'n'}>
-CONFIG_INPUT_VIVALDIFMAP                        policy<{'riscv64': 'y'}>
 CONFIG_INTEGRITY                                policy<{'riscv64': 'y'}>
 CONFIG_INTEGRITY_ASYMMETRIC_KEYS                policy<{'riscv64': 'y'}>
 CONFIG_INTEGRITY_AUDIT                          policy<{'riscv64': 'y'}>
@@ -3237,38 +3215,6 @@ CONFIG_KEXEC                                    policy<{'riscv64': 'y'}>
 CONFIG_KEXEC_CORE                               policy<{'riscv64': 'y'}>
 CONFIG_KEXEC_ELF                                policy<{'riscv64': 'y'}>
 CONFIG_KEXEC_FILE                               policy<{'riscv64': 'y'}>
-CONFIG_KEYBOARD_ADC                             policy<{'riscv64': 'n'}>
-CONFIG_KEYBOARD_ADP5588                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_ADP5589                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_ATKBD                           policy<{'riscv64': 'y'}>
-CONFIG_KEYBOARD_BCM                             policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_CAP11XX                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_CYPRESS_SF                      policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_DLINK_DIR685                    policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_GOLDFISH_EVENTS                 policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_GPIO                            policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_GPIO_POLLED                     policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_LKKBD                           policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_LM8323                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_LM8333                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_MATRIX                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_MAX7359                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_MCS                             policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_MPR121                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_NEWTON                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_OMAP4                           policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_OPENCORES                       policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_PINEPHONE                       policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_QT1050                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_QT1070                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_QT2160                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_SAMSUNG                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_STOWAWAY                        policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_SUNKBD                          policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_TCA6416                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_TCA8418                         policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_TM2_TOUCHKEY                    policy<{'riscv64': 'm'}>
-CONFIG_KEYBOARD_XTKBD                           policy<{'riscv64': 'm'}>
 CONFIG_KEYS                                     policy<{'riscv64': 'y'}>
 CONFIG_KEYS_REQUEST_CACHE                       policy<{'riscv64': 'y'}>
 CONFIG_KEY_DH_OPERATIONS                        policy<{'riscv64': 'y'}>
@@ -3860,31 +3806,6 @@ CONFIG_MODULE_UNLOAD                            policy<{'riscv64': 'y'}>
 CONFIG_MODULE_UNLOAD_TAINT_TRACKING             policy<{'riscv64': 'n'}>
 CONFIG_MOST                                     policy<{'riscv64': 'n'}>
 CONFIG_MOTORCOMM_PHY                            policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_APPLETOUCH                         policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_BCM5974                            policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_CYAPA                              policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_ELAN_I2C                           policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_ELAN_I2C_I2C                       policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_ELAN_I2C_SMBUS                     policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_GPIO                               policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_PS2                                policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_PS2_ALPS                           policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_BYD                            policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_CYPRESS                        policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_ELANTECH                       policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_ELANTECH_SMBUS                 policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_FOCALTECH                      policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_LOGIPS2PP                      policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_SENTELIC                       policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_SMBUS                          policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_SYNAPTICS                      policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_SYNAPTICS_SMBUS                policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_TOUCHKIT                       policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_PS2_TRACKPOINT                     policy<{'riscv64': 'y'}>
-CONFIG_MOUSE_SERIAL                             policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_SYNAPTICS_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_SYNAPTICS_USB                      policy<{'riscv64': 'm'}>
-CONFIG_MOUSE_VSXXXAA                            policy<{'riscv64': 'm'}>
 CONFIG_MOXTET                                   policy<{'riscv64': 'm'}>
 CONFIG_MPILIB                                   policy<{'riscv64': 'y'}>
 CONFIG_MPL115_I2C                               policy<{'riscv64': 'n'}>
@@ -4635,17 +4556,7 @@ CONFIG_NO_HZ                                    policy<{'riscv64': 'y'}>
 CONFIG_NO_HZ_COMMON                             policy<{'riscv64': 'y'}>
 CONFIG_NO_HZ_FULL                               policy<{'riscv64': 'n'}>
 CONFIG_NS83820                                  policy<{'riscv64': 'm'}>
-CONFIG_NTB                                      policy<{'riscv64': 'm'}>
-CONFIG_NTB_EPF                                  policy<{'riscv64': 'm'}>
-CONFIG_NTB_IDT                                  policy<{'riscv64': 'm'}>
-CONFIG_NTB_MSI                                  policy<{'riscv64': 'y'}>
-CONFIG_NTB_MSI_TEST                             policy<{'riscv64': 'n'}>
-CONFIG_NTB_NETDEV                               policy<{'riscv64': 'm'}>
-CONFIG_NTB_PERF                                 policy<{'riscv64': 'm'}>
-CONFIG_NTB_PINGPONG                             policy<{'riscv64': 'm'}>
-CONFIG_NTB_SWITCHTEC                            policy<{'riscv64': 'm'}>
-CONFIG_NTB_TOOL                                 policy<{'riscv64': 'm'}>
-CONFIG_NTB_TRANSPORT                            policy<{'riscv64': 'm'}>
+CONFIG_NTB                                      policy<{'riscv64': 'n'}>
 CONFIG_NTFS3_64BIT_CLUSTER                      policy<{'riscv64': 'n'}>
 CONFIG_NTFS3_FS                                 policy<{'riscv64': 'm'}>
 CONFIG_NTFS3_FS_POSIX_ACL                       policy<{'riscv64': 'y'}>
@@ -4749,6 +4660,8 @@ CONFIG_PAGE_REPORTING                           policy<{'riscv64': 'y'}>
 CONFIG_PAGE_SIZE_LESS_THAN_256KB                policy<{'riscv64': 'y'}>
 CONFIG_PAGE_SIZE_LESS_THAN_64KB                 policy<{'riscv64': 'y'}>
 CONFIG_PAGE_TABLE_CHECK                         policy<{'riscv64': 'n'}>
+CONFIG_PAHOLE_HAS_LANG_EXCLUDE                  policy<{'riscv64': 'y'}>
+CONFIG_PAHOLE_HAS_SPLIT_BTF                     policy<{'riscv64': 'y'}>
 CONFIG_PAHOLE_VERSION                           policy<{'riscv64': '125'}>
 CONFIG_PANEL                                    policy<{'riscv64': 'm'}>
 CONFIG_PANEL_CHANGE_MESSAGE                     policy<{'riscv64': 'n'}>
@@ -4857,7 +4770,6 @@ CONFIG_PCI_ENDPOINT_TEST                        policy<{'riscv64': 'n'}>
 CONFIG_PCI_EPF_MHI                              policy<{'riscv64': 'm'}>
 CONFIG_PCI_EPF_NTB                              policy<{'riscv64': 'm'}>
 CONFIG_PCI_EPF_TEST                             policy<{'riscv64': 'n'}>
-CONFIG_PCI_EPF_VNTB                             policy<{'riscv64': 'm'}>
 CONFIG_PCI_FTPCI100                             policy<{'riscv64': 'y'}>
 CONFIG_PCI_HOST_COMMON                          policy<{'riscv64': 'y'}>
 CONFIG_PCI_HOST_GENERIC                         policy<{'riscv64': 'y'}>
@@ -4911,7 +4823,7 @@ CONFIG_PHY_QCOM_USB_HS                          policy<{'riscv64': 'n'}>
 CONFIG_PHY_QCOM_USB_HSIC                        policy<{'riscv64': 'n'}>
 CONFIG_PHY_SAMSUNG_USB2                         policy<{'riscv64': 'n'}>
 CONFIG_PHY_TUSB1210                             policy<{'riscv64': 'n'}>
-CONFIG_PI433                                    policy<{'riscv64': 'm'}>
+CONFIG_PI433                                    policy<{'riscv64': 'n'}>
 CONFIG_PID_NS                                   policy<{'riscv64': 'y'}>
 CONFIG_PINCONF                                  policy<{'riscv64': 'y'}>
 CONFIG_PINCTRL                                  policy<{'riscv64': 'y'}>
@@ -5167,8 +5079,6 @@ CONFIG_REGMAP                                   policy<{'riscv64': 'y'}>
 CONFIG_REGMAP_I2C                               policy<{'riscv64': 'm'}>
 CONFIG_REGMAP_IRQ                               policy<{'riscv64': 'y'}>
 CONFIG_REGMAP_MMIO                              policy<{'riscv64': 'y'}>
-CONFIG_REGMAP_SOUNDWIRE                         policy<{'riscv64': 'm'}>
-CONFIG_REGMAP_SOUNDWIRE_MBQ                     policy<{'riscv64': 'm'}>
 CONFIG_REGMAP_SPI                               policy<{'riscv64': 'm'}>
 CONFIG_REGMAP_SPMI                              policy<{'riscv64': 'm'}>
 CONFIG_REGULATOR                                policy<{'riscv64': 'y'}>
@@ -5995,38 +5905,32 @@ CONFIG_SMSC911X                                 policy<{'riscv64': 'm'}>
 CONFIG_SMSC9420                                 policy<{'riscv64': 'm'}>
 CONFIG_SMSC_PHY                                 policy<{'riscv64': 'm'}>
 CONFIG_SM_FTL                                   policy<{'riscv64': 'm'}>
-CONFIG_SND_AC97_CODEC                           policy<{'riscv64': 'm'}>
-CONFIG_SND_AC97_POWER_SAVE                      policy<{'riscv64': 'y'}>
-CONFIG_SND_AC97_POWER_SAVE_DEFAULT              policy<{'riscv64': '0'}>
-CONFIG_SND_AD1889                               policy<{'riscv64': 'm'}>
+CONFIG_SND_AD1889                               policy<{'riscv64': 'n'}>
 CONFIG_SND_ALOOP                                policy<{'riscv64': 'm'}>
 CONFIG_SND_AMD_ACP_CONFIG                       policy<{'riscv64': 'm'}>
-CONFIG_SND_ATIIXP                               policy<{'riscv64': 'm'}>
-CONFIG_SND_ATIIXP_MODEM                         policy<{'riscv64': 'm'}>
-CONFIG_SND_ATMEL_SOC                            policy<{'riscv64': 'm'}>
-CONFIG_SND_AU8810                               policy<{'riscv64': 'm'}>
-CONFIG_SND_AU8820                               policy<{'riscv64': 'm'}>
-CONFIG_SND_AU8830                               policy<{'riscv64': 'm'}>
+CONFIG_SND_ATIIXP                               policy<{'riscv64': 'n'}>
+CONFIG_SND_ATIIXP_MODEM                         policy<{'riscv64': 'n'}>
+CONFIG_SND_ATMEL_SOC                            policy<{'riscv64': 'n'}>
+CONFIG_SND_AU8810                               policy<{'riscv64': 'n'}>
+CONFIG_SND_AU8820                               policy<{'riscv64': 'n'}>
+CONFIG_SND_AU8830                               policy<{'riscv64': 'n'}>
 CONFIG_SND_AUDIO_GRAPH_CARD                     policy<{'riscv64': 'm'}>
 CONFIG_SND_AUDIO_GRAPH_CARD2                    policy<{'riscv64': 'm'}>
 CONFIG_SND_AUDIO_GRAPH_CARD2_CUSTOM_SAMPLE      policy<{'riscv64': 'm'}>
-CONFIG_SND_AW2                                  policy<{'riscv64': 'm'}>
+CONFIG_SND_AW2                                  policy<{'riscv64': 'n'}>
 CONFIG_SND_BCD2000                              policy<{'riscv64': 'm'}>
-CONFIG_SND_BCM63XX_I2S_WHISTLER                 policy<{'riscv64': 'm'}>
-CONFIG_SND_BT87X                                policy<{'riscv64': 'm'}>
-CONFIG_SND_BT87X_OVERCLOCK                      policy<{'riscv64': 'n'}>
-CONFIG_SND_CA0106                               policy<{'riscv64': 'm'}>
-CONFIG_SND_CMIPCI                               policy<{'riscv64': 'm'}>
-CONFIG_SND_COMPRESS_OFFLOAD                     policy<{'riscv64': 'm'}>
-CONFIG_SND_CS4281                               policy<{'riscv64': 'm'}>
-CONFIG_SND_CS46XX                               policy<{'riscv64': 'm'}>
-CONFIG_SND_CS46XX_NEW_DSP                       policy<{'riscv64': 'y'}>
+CONFIG_SND_BCM63XX_I2S_WHISTLER                 policy<{'riscv64': 'n'}>
+CONFIG_SND_BT87X                                policy<{'riscv64': 'n'}>
+CONFIG_SND_CA0106                               policy<{'riscv64': 'n'}>
+CONFIG_SND_CMIPCI                               policy<{'riscv64': 'n'}>
+CONFIG_SND_CS4281                               policy<{'riscv64': 'n'}>
+CONFIG_SND_CS46XX                               policy<{'riscv64': 'n'}>
 CONFIG_SND_CTL_FAST_LOOKUP                      policy<{'riscv64': 'n'}>
 CONFIG_SND_CTL_INPUT_VALIDATION                 policy<{'riscv64': 'n'}>
 CONFIG_SND_CTL_LED                              policy<{'riscv64': 'm'}>
-CONFIG_SND_CTXFI                                policy<{'riscv64': 'm'}>
-CONFIG_SND_DARLA20                              policy<{'riscv64': 'm'}>
-CONFIG_SND_DARLA24                              policy<{'riscv64': 'm'}>
+CONFIG_SND_CTXFI                                policy<{'riscv64': 'n'}>
+CONFIG_SND_DARLA20                              policy<{'riscv64': 'n'}>
+CONFIG_SND_DARLA24                              policy<{'riscv64': 'n'}>
 CONFIG_SND_DEBUG                                policy<{'riscv64': 'n'}>
 CONFIG_SND_DESIGNWARE_I2S                       policy<{'riscv64': 'm'}>
 CONFIG_SND_DESIGNWARE_PCM                       policy<{'riscv64': 'y'}>
@@ -6034,13 +5938,13 @@ CONFIG_SND_DMAENGINE_PCM                        policy<{'riscv64': 'm'}>
 CONFIG_SND_DRIVERS                              policy<{'riscv64': 'y'}>
 CONFIG_SND_DUMMY                                policy<{'riscv64': 'm'}>
 CONFIG_SND_DYNAMIC_MINORS                       policy<{'riscv64': 'y'}>
-CONFIG_SND_ECHO3G                               policy<{'riscv64': 'm'}>
-CONFIG_SND_ENS1370                              policy<{'riscv64': 'm'}>
-CONFIG_SND_ENS1371                              policy<{'riscv64': 'm'}>
+CONFIG_SND_ECHO3G                               policy<{'riscv64': 'n'}>
+CONFIG_SND_ENS1370                              policy<{'riscv64': 'n'}>
+CONFIG_SND_ENS1371                              policy<{'riscv64': 'n'}>
 CONFIG_SND_ESWIN_DW_I2S                         policy<{'riscv64': 'm'}>
-CONFIG_SND_FM801                                policy<{'riscv64': 'm'}>
-CONFIG_SND_GINA20                               policy<{'riscv64': 'm'}>
-CONFIG_SND_GINA24                               policy<{'riscv64': 'm'}>
+CONFIG_SND_FM801                                policy<{'riscv64': 'n'}>
+CONFIG_SND_GINA20                               policy<{'riscv64': 'n'}>
+CONFIG_SND_GINA24                               policy<{'riscv64': 'n'}>
 CONFIG_SND_HDA                                  policy<{'riscv64': 'm'}>
 CONFIG_SND_HDA_CODEC_ANALOG                     policy<{'riscv64': 'm'}>
 CONFIG_SND_HDA_CODEC_CA0110                     policy<{'riscv64': 'm'}>
@@ -6069,177 +5973,151 @@ CONFIG_SND_HDA_INTEL                            policy<{'riscv64': 'm'}>
 CONFIG_SND_HDA_INTEL_HDMI_SILENT_STREAM         policy<{'riscv64': 'n'}>
 CONFIG_SND_HDA_PATCH_LOADER                     policy<{'riscv64': 'y'}>
 CONFIG_SND_HDA_PREALLOC_SIZE                    policy<{'riscv64': '64'}>
-CONFIG_SND_HDSP                                 policy<{'riscv64': 'm'}>
-CONFIG_SND_HDSPM                                policy<{'riscv64': 'm'}>
+CONFIG_SND_HDSP                                 policy<{'riscv64': 'n'}>
+CONFIG_SND_HDSPM                                policy<{'riscv64': 'n'}>
 CONFIG_SND_HRTIMER                              policy<{'riscv64': 'm'}>
 CONFIG_SND_HWDEP                                policy<{'riscv64': 'm'}>
-CONFIG_SND_I2S_HI6210_I2S                       policy<{'riscv64': 'm'}>
-CONFIG_SND_ICE1724                              policy<{'riscv64': 'm'}>
-CONFIG_SND_INDIGO                               policy<{'riscv64': 'm'}>
-CONFIG_SND_INDIGODJ                             policy<{'riscv64': 'm'}>
-CONFIG_SND_INDIGODJX                            policy<{'riscv64': 'm'}>
-CONFIG_SND_INDIGOIO                             policy<{'riscv64': 'm'}>
-CONFIG_SND_INDIGOIOX                            policy<{'riscv64': 'm'}>
-CONFIG_SND_INTEL8X0                             policy<{'riscv64': 'm'}>
-CONFIG_SND_INTEL8X0M                            policy<{'riscv64': 'm'}>
+CONFIG_SND_I2S_HI6210_I2S                       policy<{'riscv64': 'n'}>
+CONFIG_SND_ICE1724                              policy<{'riscv64': 'n'}>
+CONFIG_SND_INDIGO                               policy<{'riscv64': 'n'}>
+CONFIG_SND_INDIGODJ                             policy<{'riscv64': 'n'}>
+CONFIG_SND_INDIGODJX                            policy<{'riscv64': 'n'}>
+CONFIG_SND_INDIGOIO                             policy<{'riscv64': 'n'}>
+CONFIG_SND_INDIGOIOX                            policy<{'riscv64': 'n'}>
+CONFIG_SND_INTEL8X0                             policy<{'riscv64': 'n'}>
+CONFIG_SND_INTEL8X0M                            policy<{'riscv64': 'n'}>
 CONFIG_SND_INTEL_DSP_CONFIG                     policy<{'riscv64': 'm'}>
 CONFIG_SND_JACK                                 policy<{'riscv64': 'y'}>
 CONFIG_SND_JACK_INPUT_DEV                       policy<{'riscv64': 'y'}>
-CONFIG_SND_KORG1212                             policy<{'riscv64': 'm'}>
-CONFIG_SND_LAYLA20                              policy<{'riscv64': 'm'}>
-CONFIG_SND_LAYLA24                              policy<{'riscv64': 'm'}>
-CONFIG_SND_LOLA                                 policy<{'riscv64': 'm'}>
-CONFIG_SND_LX6464ES                             policy<{'riscv64': 'm'}>
+CONFIG_SND_KORG1212                             policy<{'riscv64': 'n'}>
+CONFIG_SND_LAYLA20                              policy<{'riscv64': 'n'}>
+CONFIG_SND_LAYLA24                              policy<{'riscv64': 'n'}>
+CONFIG_SND_LOLA                                 policy<{'riscv64': 'n'}>
+CONFIG_SND_LX6464ES                             policy<{'riscv64': 'n'}>
 CONFIG_SND_MAX_CARDS                            policy<{'riscv64': '32'}>
-CONFIG_SND_MIA                                  policy<{'riscv64': 'm'}>
-CONFIG_SND_MIXART                               policy<{'riscv64': 'm'}>
+CONFIG_SND_MIA                                  policy<{'riscv64': 'n'}>
+CONFIG_SND_MIXART                               policy<{'riscv64': 'n'}>
 CONFIG_SND_MIXER_OSS                            policy<{'riscv64': 'm'}>
-CONFIG_SND_MONA                                 policy<{'riscv64': 'm'}>
-CONFIG_SND_MPU401                               policy<{'riscv64': 'm'}>
-CONFIG_SND_MPU401_UART                          policy<{'riscv64': 'm'}>
+CONFIG_SND_MONA                                 policy<{'riscv64': 'n'}>
+CONFIG_SND_MPU401                               policy<{'riscv64': 'n'}>
 CONFIG_SND_MTPAV                                policy<{'riscv64': 'm'}>
 CONFIG_SND_MTS64                                policy<{'riscv64': 'm'}>
-CONFIG_SND_NM256                                policy<{'riscv64': 'm'}>
-CONFIG_SND_OPL3_LIB                             policy<{'riscv64': 'm'}>
-CONFIG_SND_OPL3_LIB_SEQ                         policy<{'riscv64': 'm'}>
+CONFIG_SND_NM256                                policy<{'riscv64': 'n'}>
 CONFIG_SND_OSSEMUL                              policy<{'riscv64': 'y'}>
-CONFIG_SND_OXYGEN                               policy<{'riscv64': 'm'}>
-CONFIG_SND_OXYGEN_LIB                           policy<{'riscv64': 'm'}>
+CONFIG_SND_OXYGEN                               policy<{'riscv64': 'n'}>
 CONFIG_SND_PCI                                  policy<{'riscv64': 'y'}>
 CONFIG_SND_PCM                                  policy<{'riscv64': 'm'}>
 CONFIG_SND_PCMTEST                              policy<{'riscv64': 'm'}>
 CONFIG_SND_PCM_ELD                              policy<{'riscv64': 'y'}>
 CONFIG_SND_PCM_IEC958                           policy<{'riscv64': 'y'}>
 CONFIG_SND_PCM_TIMER                            policy<{'riscv64': 'y'}>
-CONFIG_SND_PCXHR                                policy<{'riscv64': 'm'}>
-CONFIG_SND_PORTMAN2X4                           policy<{'riscv64': 'm'}>
+CONFIG_SND_PCXHR                                policy<{'riscv64': 'n'}>
+CONFIG_SND_PORTMAN2X4                           policy<{'riscv64': 'n'}>
 CONFIG_SND_PROC_FS                              policy<{'riscv64': 'y'}>
 CONFIG_SND_RAWMIDI                              policy<{'riscv64': 'm'}>
-CONFIG_SND_RIPTIDE                              policy<{'riscv64': 'm'}>
-CONFIG_SND_RME32                                policy<{'riscv64': 'm'}>
-CONFIG_SND_RME96                                policy<{'riscv64': 'm'}>
-CONFIG_SND_RME9652                              policy<{'riscv64': 'm'}>
+CONFIG_SND_RIPTIDE                              policy<{'riscv64': 'n'}>
+CONFIG_SND_RME32                                policy<{'riscv64': 'n'}>
+CONFIG_SND_RME96                                policy<{'riscv64': 'n'}>
+CONFIG_SND_RME9652                              policy<{'riscv64': 'n'}>
+CONFIG_SND_SE6X                                 policy<{'riscv64': 'n'}>
 CONFIG_SND_SEQUENCER                            policy<{'riscv64': 'm'}>
 CONFIG_SND_SEQUENCER_OSS                        policy<{'riscv64': 'n'}>
 CONFIG_SND_SEQ_DEVICE                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SEQ_DUMMY                            policy<{'riscv64': 'm'}>
 CONFIG_SND_SEQ_HRTIMER_DEFAULT                  policy<{'riscv64': 'y'}>
 CONFIG_SND_SEQ_MIDI                             policy<{'riscv64': 'm'}>
-CONFIG_SND_SEQ_MIDI_EMUL                        policy<{'riscv64': 'm'}>
 CONFIG_SND_SEQ_MIDI_EVENT                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SEQ_UMP                              policy<{'riscv64': 'y'}>
 CONFIG_SND_SEQ_UMP_CLIENT                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SEQ_VIRMIDI                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SERIAL_GENERIC                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SERIAL_U16550                        policy<{'riscv64': 'm'}>
+CONFIG_SND_SERIAL_GENERIC                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SERIAL_U16550                        policy<{'riscv64': 'n'}>
 CONFIG_SND_SIMPLE_CARD                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SIMPLE_CARD_UTILS                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AC97_BUS                         policy<{'riscv64': 'y'}>
-CONFIG_SND_SOC_AC97_CODEC                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1372                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1372_I2C                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1372_SPI                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1701                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1761                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1761_I2C                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU1761_SPI                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU17X1                         policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_AC97_CODEC                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADAU1372_I2C                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADAU1372_SPI                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADAU1701                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADAU1761_I2C                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADAU1761_SPI                     policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_ADAU7002                         policy<{'riscv64': 'n'}>
-CONFIG_SND_SOC_ADAU7118                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU7118_HW                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU7118_I2C                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADAU_UTILS                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADI                              policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADI_AXI_I2S                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ADI_AXI_SPDIF                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4104                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4118                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4375                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4458                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4554                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4613                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK4642                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK5386                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AK5558                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ALC5623                          policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_ADAU7118_HW                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADAU7118_I2C                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ADI                              policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4104                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4118                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4375                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4458                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4554                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4613                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK4642                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK5386                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AK5558                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ALC5623                          policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_AMD_ACP                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_AMD_CZ_RT5645_MACH               policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_AUDIO_IIO_AUX                    policy<{'riscv64': 'n'}>
-CONFIG_SND_SOC_AW8738                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AW88261                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AW88395                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_AW88395_LIB                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_BD28623                          policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_AW8738                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AW88261                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_AW88395                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_BD28623                          policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_BT_SCO                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CHV3_CODEC                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CHV3_I2S                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_COMPRESS                         policy<{'riscv64': 'y'}>
-CONFIG_SND_SOC_CS35L32                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L33                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L34                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L35                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L36                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L41                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L41_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L41_LIB                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L41_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L45                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L45_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L45_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L56                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L56_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L56_SDW                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L56_SHARED                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS35L56_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4234                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4265                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4270                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4271                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4271_I2C                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4271_SPI                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L42                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L42_CORE                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L42_SDW                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L51                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L51_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L52                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L56                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L73                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42L83                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42XX8                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS42XX8_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS43130                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4341                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS4349                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CS53L30                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_CX2072X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_DA7213                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_CHV3_CODEC                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CHV3_I2S                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L32                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L33                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L34                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L35                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L36                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L41_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L41_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L45_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L45_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L56_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L56_SDW                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS35L56_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4234                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4265                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4270                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4271_I2C                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4271_SPI                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L42                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L42_SDW                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L51_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L52                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L56                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L73                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42L83                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS42XX8_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS43130                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4341                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS4349                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CS53L30                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_CX2072X                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_DA7213                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_DMIC                             policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ES7134                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ES7241                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_ES7134                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ES7241                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_ES8316                           policy<{'riscv64': 'n'}>
-CONFIG_SND_SOC_ES8326                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_ES8326                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_ES8328_I2C                       policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_ES8328_SPI                       policy<{'riscv64': 'n'}>
-CONFIG_SND_SOC_FSL_ASRC                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_AUDMIX                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_EASRC                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_ESAI                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_MICFIL                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_MQS                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_RPMSG                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_SAI                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_SPDIF                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_SSI                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_UTILS                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_FSL_XCVR                         policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_FSL_ASRC                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_AUDMIX                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_ESAI                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_MICFIL                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_RPMSG                        policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_SAI                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_SPDIF                        policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_SSI                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_FSL_XCVR                         policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_GENERIC_DMAENGINE_PCM            policy<{'riscv64': 'y'}>
-CONFIG_SND_SOC_GTM601                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_GTM601                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_HDA                              policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_HDMI_CODEC                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_I2C_AND_SPI                      policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_ICS43432                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_IDT821034                        policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_IDT821034                        policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_IMG                              policy<{'riscv64': 'y'}>
 CONFIG_SND_SOC_IMG_I2S_IN                       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_IMG_I2S_OUT                      policy<{'riscv64': 'm'}>
@@ -6247,185 +6125,156 @@ CONFIG_SND_SOC_IMG_PARALLEL_OUT                 policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_IMG_PISTACHIO_INTERNAL_DAC       policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_IMG_SPDIF_IN                     policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_IMG_SPDIF_OUT                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_IMX_AUDMUX                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_INNO_RK3036                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_LPASS_MACRO_COMMON               policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_LPASS_RX_MACRO                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_LPASS_TX_MACRO                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_LPASS_VA_MACRO                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_LPASS_WSA_MACRO                  policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX9759                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98088                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98090                         policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_IMX_AUDMUX                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_INNO_RK3036                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_LPASS_RX_MACRO                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_LPASS_TX_MACRO                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_LPASS_VA_MACRO                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_LPASS_WSA_MACRO                  policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX9759                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98088                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98090                         policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_MAX98357A                        policy<{'riscv64': 'n'}>
-CONFIG_SND_SOC_MAX98363                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98373                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98373_I2C                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98373_SDW                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98388                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98390                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98396                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98504                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98520                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX9860                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX9867                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MAX98927                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MIKROE_PROTO                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MSM8916_WCD_ANALOG               policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MSM8916_WCD_DIGITAL              policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MT6351                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MT6358                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_MT6660                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_MAX98363                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98373_I2C                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98373_SDW                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98388                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98390                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98396                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98504                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98520                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX9860                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX9867                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MAX98927                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MSM8916_WCD_ANALOG               policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MSM8916_WCD_DIGITAL              policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MT6351                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MT6358                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_MT6660                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_MTK_BTCVSD                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_NAU8315                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_NAU8540                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_NAU8810                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_NAU8821                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_NAU8822                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_NAU8824                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM1681                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM1789                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM1789_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM179X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM179X_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM179X_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM186X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM186X_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM186X_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM3060                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM3060_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM3060_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM3168A                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM3168A_I2C                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM3168A_SPI                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM5102A                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM512x                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM512x_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PCM512x_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_PEB2466                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RK3328                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_NAU8315                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_NAU8540                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_NAU8810                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_NAU8821                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_NAU8822                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_NAU8824                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM1681                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM1789_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM179X_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM179X_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM186X_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM186X_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM3060_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM3060_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM3168A_I2C                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM3168A_SPI                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM5102A                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM512x_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PCM512x_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_PEB2466                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RK3328                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_RL6231                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT1017_SDCA_SDW                  policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT1308_SDW                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT1316_SDW                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT1318_SDW                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT5616                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT5631                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT5640                           policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_RT1017_SDCA_SDW                  policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT1308_SDW                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT1316_SDW                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT1318_SDW                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT5616                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT5631                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT5640                           policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_RT5645                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT5659                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT5682                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT5682_SDW                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT700                            policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT700_SDW                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT711                            policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT711_SDCA_SDW                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT711_SDW                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT712_SDCA_DMIC_SDW              policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT712_SDCA_SDW                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT715                            policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT715_SDCA_SDW                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT715_SDW                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT722_SDCA_SDW                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_RT9120                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SDW_MOCKUP                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SGTL5000                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SIGMADSP                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SIGMADSP_I2C                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SIGMADSP_REGMAP                  policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_RT5659                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT5682_SDW                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT700_SDW                        policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT711_SDCA_SDW                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT711_SDW                        policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT712_SDCA_DMIC_SDW              policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT712_SDCA_SDW                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT715_SDCA_SDW                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT715_SDW                        policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT722_SDCA_SDW                   policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_RT9120                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SDW_MOCKUP                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SGTL5000                         policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_SIMPLE_AMPLIFIER                 policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SIMPLE_MUX                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SMA1303                          policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_SMA1303                          policy<{'riscv64': 'n'}>
 CONFIG_SND_SOC_SOF_OF                           policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SOF_PCI                          policy<{'riscv64': 'm'}>
 CONFIG_SND_SOC_SOF_TOPLEVEL                     policy<{'riscv64': 'y'}>
 CONFIG_SND_SOC_SPDIF                            policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SRC4XXX                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SRC4XXX_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM2305                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM2518                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM2602                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM2602_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM2602_SPI                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM3515                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_SSM4567                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_STA32X                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_STA350                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_STI_SAS                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2552                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2562                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2764                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2770                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2780                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2781_COMLIB                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2781_FMWLIB                   policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS2781_I2C                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS5086                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS571X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS5720                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS5805M                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TAS6424                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TDA7419                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TFA9879                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TFA989X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320ADC3XXX                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320ADCX140                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC23                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC23_I2C                  policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC23_SPI                  policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC31XX                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC32X4                    policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC32X4_I2C                policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC32X4_SPI                policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC3X                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC3X_I2C                  policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TLV320AIC3X_SPI                  policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TPA6130A2                        policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TS3A227E                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TSCS42XX                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_TSCS454                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_UDA1334                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WCD938X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WCD938X_SDW                      policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WCD_CLASSH                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WCD_MBHC                         policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8510                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8523                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8524                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8580                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8711                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8728                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8731                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8731_I2C                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8731_SPI                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8737                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8741                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8750                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8753                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8770                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8776                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8782                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8804                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8804_I2C                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8804_SPI                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8903                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8904                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8940                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8960                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8961                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8962                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8974                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8978                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM8985                           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WM_ADSP                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WSA881X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WSA883X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_WSA884X                          policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_XILINX_AUDIO_FORMATTER           policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_XILINX_I2S                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_XILINX_SPDIF                     policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_XTFPGA_I2S                       policy<{'riscv64': 'm'}>
-CONFIG_SND_SOC_ZL38060                          policy<{'riscv64': 'm'}>
+CONFIG_SND_SOC_SRC4XXX_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SSM2305                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SSM2518                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SSM2602_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SSM2602_SPI                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SSM3515                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_SSM4567                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_STA32X                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_STA350                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_STI_SAS                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS2552                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS2562                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS2764                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS2770                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS2780                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS2781_I2C                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS5086                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS571X                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS5720                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS5805M                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TAS6424                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TDA7419                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TFA9879                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TFA989X                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320ADC3XXX                    policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320ADCX140                    policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC23_I2C                  policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC23_SPI                  policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC31XX                    policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC32X4_I2C                policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC32X4_SPI                policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC3X_I2C                  policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TLV320AIC3X_SPI                  policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TPA6130A2                        policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TS3A227E                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TSCS42XX                         policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_TSCS454                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_UDA1334                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WCD938X_SDW                      policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8510                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8523                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8524                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8580                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8711                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8728                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8731_I2C                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8731_SPI                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8737                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8741                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8750                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8753                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8770                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8776                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8782                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8804_I2C                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8804_SPI                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8903                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8904                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8940                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8960                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8961                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8962                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8974                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8978                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WM8985                           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WSA881X                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WSA883X                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_WSA884X                          policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_XILINX_AUDIO_FORMATTER           policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_XILINX_I2S                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_XILINX_SPDIF                     policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_XTFPGA_I2S                       policy<{'riscv64': 'n'}>
+CONFIG_SND_SOC_ZL38060                          policy<{'riscv64': 'n'}>
 CONFIG_SND_SPI                                  policy<{'riscv64': 'y'}>
 CONFIG_SND_SUPPORT_OLD_API                      policy<{'riscv64': 'y'}>
 CONFIG_SND_TEST_COMPONENT                       policy<{'riscv64': 'm'}>
@@ -6448,15 +6297,14 @@ CONFIG_SND_USB_UA101                            policy<{'riscv64': 'm'}>
 CONFIG_SND_USB_VARIAX                           policy<{'riscv64': 'm'}>
 CONFIG_SND_VERBOSE_PRINTK                       policy<{'riscv64': 'n'}>
 CONFIG_SND_VERBOSE_PROCFS                       policy<{'riscv64': 'y'}>
-CONFIG_SND_VIA82XX                              policy<{'riscv64': 'm'}>
-CONFIG_SND_VIA82XX_MODEM                        policy<{'riscv64': 'm'}>
+CONFIG_SND_VIA82XX                              policy<{'riscv64': 'n'}>
+CONFIG_SND_VIA82XX_MODEM                        policy<{'riscv64': 'n'}>
 CONFIG_SND_VIRMIDI                              policy<{'riscv64': 'm'}>
 CONFIG_SND_VIRTIO                               policy<{'riscv64': 'm'}>
-CONFIG_SND_VIRTUOSO                             policy<{'riscv64': 'm'}>
+CONFIG_SND_VIRTUOSO                             policy<{'riscv64': 'n'}>
 CONFIG_SND_VMASTER                              policy<{'riscv64': 'y'}>
-CONFIG_SND_VX222                                policy<{'riscv64': 'm'}>
-CONFIG_SND_VX_LIB                               policy<{'riscv64': 'm'}>
-CONFIG_SND_YMFPCI                               policy<{'riscv64': 'm'}>
+CONFIG_SND_VX222                                policy<{'riscv64': 'n'}>
+CONFIG_SND_YMFPCI                               policy<{'riscv64': 'n'}>
 CONFIG_SNET_VDPA                                policy<{'riscv64': 'm'}>
 CONFIG_SOCK_CGROUP_DATA                         policy<{'riscv64': 'y'}>
 CONFIG_SOCK_RX_QUEUE_MAPPING                    policy<{'riscv64': 'y'}>
@@ -7137,7 +6985,7 @@ CONFIG_USB_OHCI_HCD                             policy<{'riscv64': 'y'}>
 CONFIG_USB_OHCI_HCD_PCI                         policy<{'riscv64': 'y'}>
 CONFIG_USB_OHCI_HCD_PLATFORM                    policy<{'riscv64': 'm'}>
 CONFIG_USB_OHCI_LITTLE_ENDIAN                   policy<{'riscv64': 'y'}>
-CONFIG_USB_ONBOARD_HUB                          policy<{'riscv64': 'm'}>
+CONFIG_USB_ONBOARD_HUB                          policy<{'riscv64': 'n'}>
 CONFIG_USB_OTG_DISABLE_EXTERNAL_HUB             policy<{'riscv64': 'n'}>
 CONFIG_USB_OTG_FSM                              policy<{'riscv64': 'm'}>
 CONFIG_USB_OTG_PRODUCTLIST                      policy<{'riscv64': 'n'}>
@@ -7217,26 +7065,26 @@ CONFIG_USB_SL811_HCD_ISO                        policy<{'riscv64': 'y'}>
 CONFIG_USB_SNP_CORE                             policy<{'riscv64': 'm'}>
 CONFIG_USB_SNP_UDC_PLAT                         policy<{'riscv64': 'm'}>
 CONFIG_USB_SPEEDTOUCH                           policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE                              policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_ALAUDA                       policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_CYPRESS_ATACB                policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_DATAFAB                      policy<{'riscv64': 'm'}>
+CONFIG_USB_STORAGE                              policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_ALAUDA                       policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_CYPRESS_ATACB                policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_DATAFAB                      policy<{'riscv64': 'y'}>
 CONFIG_USB_STORAGE_DEBUG                        policy<{'riscv64': 'n'}>
-CONFIG_USB_STORAGE_ENE_UB6250                   policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_FREECOM                      policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_ISD200                       policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_JUMPSHOT                     policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_KARMA                        policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_ONETOUCH                     policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_REALTEK                      policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_SDDR09                       policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_SDDR55                       policy<{'riscv64': 'm'}>
-CONFIG_USB_STORAGE_USBAT                        policy<{'riscv64': 'm'}>
+CONFIG_USB_STORAGE_ENE_UB6250                   policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_FREECOM                      policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_ISD200                       policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_JUMPSHOT                     policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_KARMA                        policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_ONETOUCH                     policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_REALTEK                      policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_SDDR09                       policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_SDDR55                       policy<{'riscv64': 'y'}>
+CONFIG_USB_STORAGE_USBAT                        policy<{'riscv64': 'y'}>
 CONFIG_USB_SUPPORT                              policy<{'riscv64': 'y'}>
 CONFIG_USB_TEST                                 policy<{'riscv64': 'm'}>
 CONFIG_USB_TMC                                  policy<{'riscv64': 'm'}>
 CONFIG_USB_TRANCEVIBRATOR                       policy<{'riscv64': 'm'}>
-CONFIG_USB_UAS                                  policy<{'riscv64': 'm'}>
+CONFIG_USB_UAS                                  policy<{'riscv64': 'y'}>
 CONFIG_USB_UEAGLEATM                            policy<{'riscv64': 'm'}>
 CONFIG_USB_ULPI_BUS                             policy<{'riscv64': 'm'}>
 CONFIG_USB_USBNET                               policy<{'riscv64': 'm'}>
@@ -7426,10 +7274,7 @@ CONFIG_VLAN_8021Q                               policy<{'riscv64': 'm'}>
 CONFIG_VLAN_8021Q_GVRP                          policy<{'riscv64': 'y'}>
 CONFIG_VLAN_8021Q_MVRP                          policy<{'riscv64': 'y'}>
 CONFIG_VMAP_STACK                               policy<{'riscv64': 'y'}>
-CONFIG_VME_BUS                                  policy<{'riscv64': 'y'}>
-CONFIG_VME_FAKE                                 policy<{'riscv64': 'm'}>
-CONFIG_VME_TSI148                               policy<{'riscv64': 'm'}>
-CONFIG_VME_USER                                 policy<{'riscv64': 'm'}>
+CONFIG_VME_BUS                                  policy<{'riscv64': 'n'}>
 CONFIG_VMLINUX_MAP                              policy<{'riscv64': 'y'}>
 CONFIG_VMXNET3                                  policy<{'riscv64': 'm'}>
 CONFIG_VM_EVENT_COUNTERS                        policy<{'riscv64': 'y'}>
@@ -7574,7 +7419,7 @@ CONFIG_XILLYBUS_CLASS                           policy<{'riscv64': 'm'}>
 CONFIG_XILLYBUS_OF                              policy<{'riscv64': 'm'}>
 CONFIG_XILLYBUS_PCIE                            policy<{'riscv64': 'm'}>
 CONFIG_XILLYUSB                                 policy<{'riscv64': 'm'}>
-CONFIG_XIL_AXIS_FIFO                            policy<{'riscv64': 'm'}>
+CONFIG_XIL_AXIS_FIFO                            policy<{'riscv64': 'n'}>
 CONFIG_XOR_BLOCKS                               policy<{'riscv64': 'm'}>
 CONFIG_XPS                                      policy<{'riscv64': 'y'}>
 CONFIG_XXHASH                                   policy<{'riscv64': 'y'}>
-- 
2.47.0

